pkg/defkinds/nodejs/builder.go

```diff
@@ -189,14 +189,15 @@ func setImageMetadata(stageDef StageDefinition, state llb.State, img *image.Imag
 		img.Config.Volumes[fullpath] = struct{}{}
 	}
 
-	if *stageDef.Healthcheck {
+	// @TODO: improve
```

Comment:
is it blocking right now or should it be done in an other PR ?

Reply:
It's done in the PR dedicated to healthcheck configuration :wink: